helm/examples/azure/sa/list-sa.sh

```diff
@@ -1,13 +1,9 @@
 #!/bin/bash
 WORK_DIR=$(dirname $0)
 source ${WORK_DIR}/env-sa
-source ${WORK_DIR}/common-libs
 
 VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
-get_share_names
-[[ -n "$SHARE_NAMES_INPUT" ]] && SHARE_NAMES=${SHARE_NAMES_INPUT}
-SHARE_NAMES=$(echo $SHARE_NAMES | sed 's/,/ /g' | tr -s ' ' | sed 's/^ $//g'  )
 
 # list share
-account_key=$(cat ${SA_KEY_DIR}/${SHARE_NAME}.key | cut -d':' -f2 | sed 's/[[:space:]]\+//g')
+account_key=$(cat ${SA_KEY_DIR}/${STORAGE_ACCOUNT_NAME}.key | cut -d':' -f2 | sed 's/[[:space:]]\+//g')
```

Comment:
Is 's/[[:space:]]\\+//g' correct?  You're searching for a + symbol?

Reply:
In bash 5.0.17  sed 4.7.
Sample code:
```code
s="dafa  dee"
echo $s
echo  $s | sed 's/[[:space:]]\+/---/g'
```
Result:
```console
dafa dee
dafa---dee
```

Without '\' can't match

Reply:
Right!  I forgot that we're dealing with bash.  Apologies.